private/buf/cmd/buf/buf.go

```diff
@@ -151,20 +151,20 @@ func NewRootCommand(name string) *appcmd.Command {
 			},
 			{
 				Use:        "mod",
-				Short:      `Manage Buf modules. All commands are deprecated and have moved to the "buf config", "buf dep", or "buf registry" subcommands.`,
-				Deprecated: `All commands are deprecated and have moved to the "buf config", "buf dep", or "buf registry" subcommands.`,
+				Short:      `Manage Buf modules, all commands are deprecated and have moved to the "buf config", "buf dep", or "buf registry" subcommands.`,
+				Deprecated: `all commands are deprecated and have moved to the "buf config", "buf dep", or "buf registry" subcommands.`,
```

Comment:
Uses the lowercase to allow building the string message with `,`:

```
$ buf mod
Command "mod" is deprecated, all commands are deprecated and have moved to the "buf config", "buf dep", or "buf registry" subcommands.
...
```

The short message above was altered to match althought it's never displayed as `Hidden: true` is set.